lib/src/third_party/spdx/licenses/LGPL-3.0.txt

```diff
@@ -69,3 +69,236 @@ Each version is given a distinguishing version number. If the Library as you rec
 
 If the Library as you received it specifies that a proxy can decide whether future versions of the GNU Lesser General Public License shall
 apply, that proxy's public statement of acceptance of any version is permanent authorization for you to choose that version for the Library.
+
+GNU GENERAL PUBLIC LICENSE
```

Comment:
including the GPL in the LGPL is optional: https://spdx.org/licenses/LGPL-3.0-or-later.html

however this is not modeled by this code, so the similarity goes way down